File: kits/message/Message.go
package message

type Message struct {
    CloudEventsVersion 	string		`json:"cloudEventsVersion"`
    EventType		string		`json:"eventType"`
    EventTypeVersion 	string		`json:"eventTypeVersion"`
    Source 		string		`json:"source"`
    EventID 		string		`json:"eventID"`
    EventTime 		string		`json:"eventTime"`
    Extensions 		[]string 	`json:"extensions"`
    ContentType 	string 		`json:"contentType"`
    Data 		[]string	`json:"data"`
}

File: kits/message/message.js
'use strict';

var Message = {
    cloudEventsVersion : "0.1",
    eventType : "com.example.someevent",
    eventTypeVersion : "1.0",
    source : "/mycontext",
    eventID : "C234-1234-1234",
    eventTime : "2018-04-05T17:31:00Z",
    extensions : {
      comExampleExtension : "value"
    },
    contentType : "application/json",
    data : {}
};
